src/lib/mongoBackend/mongoSubscribeContext.cpp

```diff
@@ -222,10 +223,15 @@ HttpStatusCode mongoSubscribeContext
 
     LM_T(LmtSubCache, ("inserting a new sub in cache (%s)", oidString.c_str()));
 
+    // FIXME Px: In the near future, the subscription carries the HttpInfo
```

Comment:
NGSIv1 wouldn't support any of the fields in httpInfo, except url. Thus, I guess that with the wrapping in line 229 is enought.

I mean, all ok, except this comment :) that should be removed.